CHANGELOG.md

```diff
@@ -6,12 +6,20 @@ All notable changes to Wallaroo will be documented in this file.
 
 ### Fixed
 
+- Gradually back off when attempting to reconnect on data channel
+- Never use more workers than there are partitions
+- No longer treat state comp stages as a special case
```

Comment:
"No longer treat state computation stages as a special case.  This results in fewer allocations and better performance."